test/unit/hubAuth.test.ts

```diff
@@ -1,16 +1,164 @@
 /*
- * Copyright (c) 2020, salesforce.com, inc.
+ * Copyright (c) 2021, salesforce.com, inc.
```

Comment:
I did not write unit test for testkitHubAuth. The key element to test in that function are the callouts to each `sfdx` auth commands, which I would need to stub out.